tools/stringtable_validator.py

```diff
@@ -146,9 +146,11 @@ def main():
     print("-----------------------")
 
     # Allow running from root directory and tools directory
-    root_dir = ".."
-    if os.path.exists("addons"):
-        root_dir = "."
+    for folder in ['addons', 'optionals']:
+        # Allow running from root directory as well as from inside the tools directory
+        root_dir = "../" + folder
+        if (os.path.exists(folder)):
+            root_dir = folder
```

Comment:
I don't think this is needed. Stringtable validator already validated all `stringtable.xml` files in the project. Now, in GitHub Actions it is only checking `stringtable.xml` files in `optionals`.

Reply:
Fixed